README.md

```diff
@@ -46,6 +46,18 @@ A fixture file is available, you can import its data with :
 $ yarn fixtures
 ```
 
+You can also import more than 2000 contacts in one batch with this other command :
+```
```

Comment:
Can you add a line break before \`\`\`, it ensures that the markdown parser from the docs generator understands correctly the code block (not the same parser as GitHub).